client/src/components/pages/FilterPage/style.css

```
@@ -0,0 +1,34 @@
+.Filter-page__container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.filter__form {
+    width: 90%;
+}
+
+.filter__title,
+.card__title
+ {
+    font-family: Sansation;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 24px;
+    line-height: 30px;
+    width: 100%;
+    color: #242424;
+    margin-left:20px; 
```

```suggestion
    margin-left: 20px; 
```
install CSS Formatter, it will help you